src/ui/Calendar.scss

```diff
@@ -0,0 +1,6 @@
+.widget-calendar {
+    .rbc-calendar {
+        min-height: 580px !important;
+        height: 580px;
```

Comment:
we should have height settings, same as the current googlemaps, charts etc
See https://github.com/mendixlabs/charts/blob/51a6114ef145d2f933d15a65c42e567e00808643/src/HeatMap/HeatMap.xml#L223-L251

Reply:
Done